gtsfm/data_association/point3d_initializer.py

```diff
@@ -159,20 +175,22 @@ def triangulate(self, track_2d: SfmTrack2d) -> Optional[SfmTrack]:
             )
         except RuntimeError:
             logger.info("Cheirality exception from GTSAM's triangulatePoint3() likely due to outlier, skipping track")
-            return None
+            return None, None
 
         # compute reprojection errors for each measurement
         reproj_errors = self.compute_track_reprojection_errors(inlier_track.measurements, triangulated_pt)
 
         # all the measurements should have error < threshold
         if not np.all(reproj_errors < self.reproj_error_thresh):
-            return None
+            return None, None
```

Comment:
Since we have reprojection errors, cant we return the mean here?

Reply:
That's a good point. I was a bit torn whether we should dump statistics of *accepted* tracks, or of any *generated* track, even if rejected.

@ayushbaid any thoughts here? maybe we should dump both for completeness now, since DA's failures are a bit of a mystery?

Reply:
Cool, I added these extra metrics now